Resources/doc/serialization-groups-and-relations.md

```diff
@@ -0,0 +1,160 @@
+# Serialization groups and relations
+
+## Using serialization groups
+
+Symfony 2.7 introduced [serialization (and deserialization) groups support](http://symfony.com/blog/new-in-symfony-2-7-serialization-groups)
+in the Serializer component. Specifying to the API system the groups to use is damn easy:
+
+```yaml
+services:
+    resource.product:
+        parent:    "api.resource"
+        arguments: [ "AppBundle\Entity\Product" ]
+        calls:
+            -      method:    "initNormalizationContext"
+                   arguments:
+                       -      { groups: [ "serialization_group1", "serialization_group2" ] }
```

Comment:
Is there any reason to keep the `groups` key and not simply pass the array like this:

```yml
method: "initNormalizationContext"
calls:
    -   method:    "initNormalizationContext"
        arguments: [ [ "serialization_group1", "serialization_group2" ] ]
```
?